permissions/Sharing.permission

```diff
@@ -32,6 +32,11 @@ dbus-user.broadcast com.jolla.PinQuery=com.jolla.PinQuery.*@/*
 dbus-user.talk com.jolla.Connectiond
 dbus-user.broadcast com.jolla.Connectiond=com.jolla.Connectiond.*@/*
 
+# Sharing process
+whitelist /usr/share/sailfish-share
```

Comment:
Could you elaborate on what is happening in this directory?

Reply:
For the sharing dialog UI don't think it belongs here. First as "app specific" thing it could be more of a .profile, but then again as system UI not sure do we even want to sandbox it.

Reply:
Ah yes, this is longer necessary as sandboxing was removed elsewhere. Updated to remove this now.